dev/ansible/00_create_intention_allow_mc_minio.yml

```diff
@@ -1,6 +1,10 @@
+# We need this intention allow to be created from mc to minio before the terraform apply code is run so as to prevent
+# the mc nomad job from failing(when consul acl default policy is set to deny). The mc nomad job creates buckets in minio
+# and it fails at that stage if the explicit allow intentions are not already in place.
```

Comment:
I think in the future we need to decouple `minio-client` from a module or build own standalone imag (minio server + minio client)